Rocket.Chat/Managers/PushManager.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PushManager.swift
+//  Rocket.Chat
+//
+//  Created by Gradler Kim on 2017. 1. 23..
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+class PushManager {
```

Comment:
What about making it `final`?
We would benefit from compilers optimizations and use static dispatch!